docs/integrations/google/workspace/collect-logs.md

```diff
@@ -18,19 +18,21 @@ This procedure explains how to collect logs from Google Workspace and ingest the
 **Google Workspace Alert Center** alerts are in JSON format. Most of the alerts have a few common fields. The differences are in the data section of the JSON where the alert type specific details are recorded. For more information, see this Google Workspace [Alert document](https://developers.google.com/admin-sdk/alertcenter/reference/alert-types).
 
 
-## Configure Collection for Google Workspace Source
+## Configure Collection for Google Workspace Audit Source
 
-This section provides instructions for configuring log collection for Google Workspace Source.
+This section provides instructions for configuring log collection for Google Workspace with Audit Source.
 
-[Google Workspace](#configure-collection-for-google-workspace-source): Monitors and analyzes the activity across all the Google Workspace Apps in one place. You can configure collection for each Google App for which you want to analyze events:
+[Google Workspace](#configure-collection-for-google-workspace-audit-source): Monitors and analyzes the activity across all the Google Workspace Apps in one place. You can configure collection for each Google App for which you want to analyze events:
     * Google Admin
     * Google Drive
     * Google Login
     * Google Token
 
 ### About Source Configuration  
 
-Configure one [Google Workspace Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-source) for each Google App from which you want to collect events:
+Currently, the source name for Google Workspace is still **G Suite Apps Audit Source**, which will be changed/updated shortly.
+
+Configure one [G Suite Apps Audit Source](/docs/send-data/hosted-collectors/google-source/g-suite-apps-audit-source) for each Google App from which you want to collect events:
```

Comment:
Change to "Google Workspace Apps Audit Source"?

Reply:
@duchatran Please be specific. There are several references to sources in lines 33 - 35. Please confirm which to change to "Google Workspace Apps Audit Source". Also, do any of the paths in links needs updating?

Reply:
Just change the text "G Suite Apps Audit Source".